### ISSUE TITLE
Can I use this tool for 3D voxel?

### ISSUE THREAD

**Issue by moshicaixi** [Project-MONAI/MONAI#3439]
Can I use this tool for 3D voxel segmentation. I think 3D voxel is similar to 3D medical image in structure, is that right? Hoping for your reply and advice.

**Comment by Nic-Ma** [comment]
Hi @moshicaixi ,

Thanks for your interest here.
@ericspod @wyli @dongyang0122 Could you please help share more info here?

Thanks in advance.

**Comment by wyli** [comment]
Yes, it should be feasible. I'm converting this to a discussion.
